bundle/manifests/idp-mgmt-operator.clusterserviceversion.yaml

```diff
@@ -535,5 +535,4 @@ spec:
   provider:
     name: Red Hat
     url: https://github.com/identitatem
-  replaces: idp-mgmt-operator.v0.2.0
```

Comment:
Please revert these changes.

Reply:
Current `make bundle` did this.   I added more to README to help remind us to revert.